client/jetpack-cloud/sections/partner-portal/update-company-details-form/index.tsx

```diff
@@ -22,7 +22,7 @@ export default function UpdateCompanyDetailsForm() {
 	const contactPerson = partner?.contact_person ?? '';
 	const companyWebsite = partner?.company_website ?? '';
 	const companyType = partner?.company_type ?? '';
-	const managedSites = partner?.managed_sites ?? '';
+	const managedSites = partner?.managed_sites ?? '1-5';
```

Comment:
It seems that the `getCurrentPartner` is not retrieving the current `managed_sites` so it's overwriting that value in the server.

Reply:
I've updated the GET partner/ API endpoint, and it works as expected

Reply:
Also, if the `managed_sites` is null, it will be overwritten by "1-5", introducing a fake data.

Reply:
Better if we don't set the default value here and leave it as `null`, that is a valid input.

Reply:
Thanks for highlighting that, I've added a new commit changing this to null now.